### PR TITLE
implement logos page

### PR DESCRIPTION
**Pull request by mansona**
Fixes https://github.com/ember-learn/ember-website/issues/441

### CODE REVIEW

**Comment by locks** [comment]
It's slightly odd that the icons aren't centered on narrower widths. Could this be tweaked?

File: app/styles/app.css [hunk 1]
```diff
@@ -15,3 +15,23 @@
 .align-middle {
   vertical-align: middle;
 }
+
+.brand-logos > li {
```
Comment (MelSumner) [inline app/styles/app.css]: shouldn't this get moved to styleguide?

File: app/styles/app.css [hunk 2]
```diff
@@ -15,3 +15,23 @@
...
+  border: 1px solid rgba($black, .3);
+  background-image: url("/images/brand/brand-bg.png");
+  border-radius: 4px;
+  padding: 10px 30px;
```
Comment (MelSumner) [inline app/styles/app.css]: this needs to be relational units not pixels

File: app/templates/logos.hbs [hunk 3]
```diff
@@ -1,175 +1,192 @@
...
-  <ul class="brand-logos">
-    <li><img alt="Tomster lockup dark background" src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img alt="Tomster lockup light background" src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img alt="Tomster lockup red background" src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img alt="Tomster lockup transparent background" src="/images/brand/ember_Tomster-Lockup.png"></li>
-  </ul>
-  <p>A fun alternative to the primary logotype is the Tomster lockup. It’s designed with a white stroke which allows it to work on all background colors without modification.</p>
-</section>
+  <section>
+    <h3 class="mt-4">Tomster Lockup</h3>
```
Comment (MelSumner) [inline app/templates/logos.hbs]:
```suggestion
    <h3 class="mt-4" id="tomster-lockup">Tomster Lockup</h3>
```

File: app/templates/logos.hbs [hunk 4]
```diff
@@ -1,175 +1,192 @@
...
-  <h3>Tomster Lockup</h3>
-  <ul class="brand-logos">
-    <li><img alt="Tomster lockup dark background" src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img alt="Tomster lockup light background" src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img alt="Tomster lockup red background" src="/images/brand/ember_Tomster-Lockup.png"></li>
-    <li><img alt="Tomster lockup transparent background" src="/images/brand/ember_Tomster-Lockup.png"></li>
-  </ul>
-  <p>A fun alternative to the primary logotype is the Tomster lockup. It’s designed with a white stroke which allows it to work on all background colors without modification.</p>
-</section>
+  <section>
```
Comment (MelSumner) [inline app/templates/logos.hbs]:
```suggestion
  <section aria-labelledby="tomster-lockup">
```

File: app/templates/logos.hbs [hunk 5]
```diff
@@ -1,175 +1,192 @@
...
+        <li><img alt="Tomster lockup transparent background" src="/images/brand/ember_Tomster-Lockup.png"></li>
+      </ul>
+    </div>
+    <p>A fun alternative to the primary logotype is the Tomster lockup. It’s designed with a white stroke which allows it to work on all background colors without modification.</p>
+  </section>
 
-<section>
-  <h3>Icon</h3>
+  <section>
+    <h3 class="mt-4">Icon</h3>
```
Comment (MelSumner) [inline app/templates/logos.hbs]:
```suggestion
    <h3 class="mt-4" id="branding-icon">Icon</h3>
```

File: app/templates/logos.hbs [hunk 6]
```diff
@@ -1,175 +1,192 @@
...
+        <li><img alt="Tomster lockup red background" src="/images/brand/ember_Tomster-Lockup.png"></li>
+        <li><img alt="Tomster lockup transparent background" src="/images/brand/ember_Tomster-Lockup.png"></li>
+      </ul>
+    </div>
+    <p>A fun alternative to the primary logotype is the Tomster lockup. It’s designed with a white stroke which allows it to work on all background colors without modification.</p>
+  </section>
 
-<section>
-  <h3>Icon</h3>
+  <section>
```
Comment (MelSumner) [inline app/templates/logos.hbs]:
```suggestion
  <section aria-labelledby="branding-icon">
```

File: app/templates/logos.hbs [hunk 7]
```diff
@@ -1,175 +1,192 @@
...
-    <li><img alt="Ember Data logo dark background" src="/images/brand/ember_Data-Dark.png"></li>
-    <li><img alt="Ember Data logo light background" src="/images/brand/ember_Data-Light.png"></li>
-    <li><img alt="Ember Data logo red background white font" src="/images/brand/ember_Data-1c-Light.png"></li>
-    <li><img alt="Ember Data logo black font" src="/images/brand/ember_Data-1c-Dark.png"></li>
-  </ul>
 
-  <p>The Ember project has numerous projects under its umbrella. These other logos may be used under the same terms as the primary Ember logo.</p>
-</section>
+  <section>
+    <h3 class="mt-4">Subproject Logos</h3>
```
Comment (MelSumner) [inline app/templates/logos.hbs]:
```suggestion
    <h3 class="mt-4" id="subproject-logos">Subproject Logos</h3>
```

File: app/templates/logos.hbs [hunk 8]
```diff
@@ -1,175 +1,192 @@
...
-  <ul class="brand-logos">
-    <li><img alt="Ember Data logo dark background" src="/images/brand/ember_Data-Dark.png"></li>
-    <li><img alt="Ember Data logo light background" src="/images/brand/ember_Data-Light.png"></li>
-    <li><img alt="Ember Data logo red background white font" src="/images/brand/ember_Data-1c-Light.png"></li>
-    <li><img alt="Ember Data logo black font" src="/images/brand/ember_Data-1c-Dark.png"></li>
-  </ul>
 
-  <p>The Ember project has numerous projects under its umbrella. These other logos may be used under the same terms as the primary Ember logo.</p>
-</section>
+  <section>
```
Comment (MelSumner) [inline app/templates/logos.hbs]:
```suggestion
  <section aria-labelledby="subproject-logos">
```

File: app/templates/logos.hbs [hunk 9]
```diff
@@ -1,175 +1,192 @@
...
-  </ul>
 
-  <ul class="brand-logos">
-    <li><img alt="Ember Times logo dark background" src="/images/brand/ember_Times-Dark.png"></li>
-    <li><img alt="Ember Times logo light background" src="/images/brand/ember_Times-Light.png"></li>
-    <li><img alt="Ember Times red background white font" src="/images/brand/ember_Times-1c-Light.png"></li>
-    <li><img alt="Ember Times black font" src="/images/brand/ember_Times-1c-Dark.png"></li>
-  </ul>
+  <section>
+    <h3 class="mt-4">Tertiary Logos</h3>
```
Comment (MelSumner) [inline app/templates/logos.hbs]:
```suggestion
    <h3 class="mt-4" id="tertiary-logos">Tertiary Logos</h3>
```

File: app/templates/logos.hbs [hunk 10]
```diff
@@ -1,175 +1,192 @@
...
-    <li><img alt="Ember Forum logo black font" src="/images/brand/ember_Forum-1c-Dark.png"></li>
-  </ul>
 
-  <ul class="brand-logos">
-    <li><img alt="Ember Times logo dark background" src="/images/brand/ember_Times-Dark.png"></li>
-    <li><img alt="Ember Times logo light background" src="/images/brand/ember_Times-Light.png"></li>
-    <li><img alt="Ember Times red background white font" src="/images/brand/ember_Times-1c-Light.png"></li>
-    <li><img alt="Ember Times black font" src="/images/brand/ember_Times-1c-Dark.png"></li>
-  </ul>
+  <section>
```
Comment (MelSumner) [inline app/templates/logos.hbs]:
```suggestion
  <section aria-labelledby="tertiary-logos">
```

File: app/templates/logos.hbs [hunk 11]
```diff
@@ -1,175 +1,192 @@
...
 
-<p>
-  The Ember brand and associated assets are part of a meticulously curated experience. We take great pride in what our team and community have helped bring to life, and ask that you follow our guidelines to ensure consistency across all Ember properties.
-</p>
+  <p>
+    The Ember brand and associated assets are part of a meticulously curated experience. We take great pride in what our team and community have helped bring to life, and ask that you follow our guidelines to ensure consistency across all Ember properties.
+  </p>
 
-<p><a href="/images/brand/Ember-Brand-Guidelines.pdf">Download brand guidelines (PDF)</a></p>
+  <p><a href="/images/brand/Ember-Brand-Guidelines.pdf">Download brand guidelines (PDF)</a></p>
```
Comment (MelSumner) [inline app/templates/logos.hbs]: Is there a `<section>` that should wrap this h2+content? There is a `<section>` element that wraps the next h2.

File: app/templates/logos.hbs [hunk 12]
```diff
@@ -1,175 +1,192 @@
...
-  <p>Ember communities and smaller projects have logos based on Ember’s <span class="code">code</span> style. They are only to be used by the project owners themselves the official properties.</p>
-</section>
+    <p>Ember communities and smaller projects have logos based on Ember’s <span class="code">code</span> style. They are only to be used by the project owners themselves the official properties.</p>
+  </section>
 
 
-<section>
-  <h2>The Ember Mascots</h2>
+  <section>
+    <h2 class="mt-4">The Ember Mascots</h2>
```
Comment (MelSumner) [inline app/templates/logos.hbs]:
```suggestion
    <h2 class="mt-4" id="ember-mascots">The Ember Mascots</h2>
```

File: app/templates/logos.hbs [hunk 13]
```diff
@@ -1,175 +1,192 @@
...
 
-  <p>Ember communities and smaller projects have logos based on Ember’s <span class="code">code</span> style. They are only to be used by the project owners themselves the official properties.</p>
-</section>
+    <p>Ember communities and smaller projects have logos based on Ember’s <span class="code">code</span> style. They are only to be used by the project owners themselves the official properties.</p>
+  </section>
 
 
-<section>
-  <h2>The Ember Mascots</h2>
+  <section>
```
Comment (MelSumner) [inline app/templates/logos.hbs]:
```suggestion
  <section aria-labelledby="ember-mascots">
```

File: app/templates/logos.hbs [hunk 14]
```diff
@@ -1,175 +1,192 @@
...
 
-  <p>
-    We sometimes grant licenses for specific use cases, and certain community initiatives may, with approval, commission the design of custom Tomsters and Zoeys via official channels. For more info on commissioned mascots, {{#link-to "mascots.faq"}}visit the Tomster FAQ page{{/link-to}}.
-  </p>
+    <p>
+      We sometimes grant licenses for specific use cases, and certain community initiatives may, with approval, commission the design of custom Tomsters and Zoeys via official channels. For more info on commissioned mascots, {{#link-to "mascots.faq"}}visit the Tomster FAQ page{{/link-to}}.
+    </p>
 
-  <h2>Naming</h2>
+    <h2>Naming</h2>
```
Comment (MelSumner) [inline app/templates/logos.hbs]:
```suggestion
</section>
<section aria-labelledby="project-naming">
    <h2 id="project-naming">Naming</h2>
```

File: app/templates/logos.hbs (hunk 11 above)
Comment (mansona) [reply app/templates/logos.hbs]: I have wrapped it in a section and added the aria-labelledby 👍

File: app/styles/app.css (hunk 2 above)
Comment (mansona) [reply app/styles/app.css]: Fixed 👍

File: app/styles/app.css (hunk 1 above)
Comment (mansona) [reply app/styles/app.css]: So... no, not really. This is a very specific style that will only be used on the website (and really only this page). Ideally I would have turned the "brand logos grid" into a component but I was being a bit lazy here in the interest of speed.

I have moved the styles to a `brand-logos.css` to make it clear that it is a component's style even without converting the html to a component. Are you ok with it now? Do you think I would need to actually create the component to do this?

**Pull request merged**